modules/EvseManager/EvseManager.cpp

```diff
@@ -350,21 +366,26 @@ void EvseManager::ready() {
         });
 
         // implement Auth handlers
-        r_hlc[0]->subscribe_EVCCIDD([this](const std::string& _token) {
-            std::string token = _token;
-            token.erase(remove(token.begin(), token.end(), ':'), token.end());
-            autocharge_token.id_token = "VID:" + token;
-            autocharge_token.type = types::authorization::TokenType::Autocharge;
-            std::vector<int> referenced_connectors = {this->config.connector_id};
-            autocharge_token.connectors.emplace(referenced_connectors);
+        if (!config.autocharge_use_slac_instead_of_hlc) {
+            r_hlc[0]->subscribe_EVCCIDD([this](const std::string& _token) {
+                types::authorization::ProvidedIdToken autocharge_token;
+
+                std::string token = _token;
+                token.erase(remove(token.begin(), token.end(), ':'), token.end());
+                autocharge_token.id_token = "VID:" + token;
+                autocharge_token.type = types::authorization::TokenType::Autocharge;
+                autocharge_token.connectors.emplace(config.connector_id, 1);
+
+                p_token_provider->publish_provided_token(autocharge_token);
```

Comment:
Was changed by the Plug'n Charge commit and is no longer executed here!

Reply:
should be fixed now

Reply:
👍